content-sample/src/main/content/jcr_root/etc/demandware/.content.xml

```diff
@@ -21,9 +21,10 @@
                 jcr:primaryType="nt:unstructured"
```

Comment:
what's that file for?

Reply:
it creates a page with "Demandware Instance initializer"

Reply:
move ti sample-content